code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -241,16 +241,22 @@
 	retreat_distance = 3
 	minimum_distance = 3
 	pass_flags = PASSTABLE
-
+	var/recentlyspawned = 0
+// This code denotes how often the hivelord can spawn a brood, modify the below spawn(30) timer to change spawn time. currently set to every 3 seconds. Current spawn rate gives max of 3 broodlings per actively attacking hivelord - EchoFade
 /mob/living/simple_animal/hostile/asteroid/hivelord/OpenFire(var/the_target)
-	var/mob/living/simple_animal/hostile/asteroid/hivelordbrood/A = new /mob/living/simple_animal/hostile/asteroid/hivelordbrood(src.loc)
-	A.GiveTarget(target)
-	A.friends = friends
-	A.faction = faction
```

Comment:
Question though. Doesn't this mean, lets say that the Hivelords faction gets changed to syndicate, that the broods wouldn't spawn with the syndicate faction?